v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/SourceDbToSpanner.java

```diff
@@ -103,12 +116,33 @@ static PipelineResult run(SourceDbToSpannerOptions options) {
     Pipeline pipeline = Pipeline.create(options);
     Map<String, Set<String>> columnsToIgnore = getColumnsToIgnore(options);
     Map<String, String> tableVsPartitionMap = getTablesVsPartitionColumn(options);
+    // Ingest session file into schema object.
+    Schema schema = SessionFileReader.read(options.getSessionFilePath());
+
+    SpannerConfig spannerConfig =
+        SpannerConfig.create()
+            .withProjectId(ValueProvider.StaticValueProvider.of(options.getProjectId()))
+            .withHost(ValueProvider.StaticValueProvider.of(options.getSpannerHost()))
+            .withInstanceId(ValueProvider.StaticValueProvider.of(options.getInstanceId()))
+            .withDatabaseId(ValueProvider.StaticValueProvider.of(options.getDatabaseId()));
+
+    PCollection<Ddl> ddl =
+        pipeline.apply("Process Information Schema", new ProcessInformationSchema(spannerConfig));
+    PCollectionView<Ddl> ddlView = ddl.apply("Cloud Spanner DDL as view", View.asSingleton());
```

Comment:
During some investigation for forward migration, we figured that this Ddl is not needed as side-input. It can just be made a member of the DoFn in which it will be used. It is serializable so, should be okay to add as member.